internal/installcfg/installcfg.go

```diff
@@ -178,6 +178,14 @@ type ImageContentSource struct {
 	Source  string   `json:"source"`
 }
 
+type ImageDigestSource struct {
```

Comment:
Can we remove the ImageContentSource struct now? (line 176)

Reply:
So it seems that for now we should still keep it in installConfig struct and marked as Deprecated to match the installer https://github.com/openshift/installer/blob/master/pkg/types/installconfig.go#L148, similar to how other deprecated fields are handled. The field is planned for removal in 4.17.

Reply:
Oh ok, make sense. So can we just add a comment on the struct that it's deprecated?

Reply:
Sure. Added a comment.